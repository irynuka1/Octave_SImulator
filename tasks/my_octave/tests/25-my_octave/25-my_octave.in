L 4 4
-5096 -3370 9683 -4202
4691 5174 -6995 -5138
7837 3280 3799 -5077
-5437 2082 9277 -6046
L 4 4
-1181 -6851 6457 4798
-4198 -4266 402 -7938
8102 700 9478 4369
-8402 7949 -3750 -1119
L 8 8
9337 1500 8956 1383 -4057 5756 6078 5085
-7691 4364 -9570 9451 7485 3759 7147 8147
-4493 -125 -1220 -8673 4136 -7189 -5140 -5173
4357 6732 1101 -5697 3452 3029 -3540 4418
-532 2134 8775 -1852 -8683 -2465 -4854 -8686
5103 3251 1755 -8357 1533 -7767 -6508 2802
7481 6834 -6678 -3853 8605 8495 4362 -2035
-8540 6755 -8088 7632 9192 -6373 1227 2302
L 8 8
9821 6178 5238 2740 -7229 -3844 -4603 -1420
-393 -9745 -5902 -9133 5408 476 1667 3630
2174 8510 -7493 8524 7373 969 1819 4680
-7699 -4335 2553 5722 -5223 -9886 -7276 3457
-769 7636 -9724 -3026 -850 8589 -2190 -379
-4061 5006 8065 -3988 -8051 2354 2185 41
6794 5042 -5457 1270 -1317 -8188 -3230 7410
-7245 8645 6207 4748 9161 -1307 8420 -213
S 0 1
P 4
D 4
S 2 3
P 5
D 5
Q