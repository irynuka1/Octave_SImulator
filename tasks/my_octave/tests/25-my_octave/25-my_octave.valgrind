==12796== Memcheck, a memory error detector
==12796== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==12796== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==12796== Command: /home/irynuka1/Desktop/Programs/Tema2_1/check_my_octave_v1.1/check_my_octave/my_octave
==12796== Parent PID: 12795
==12796== 
==12796== 
==12796== HEAP SUMMARY:
==12796==     in use at exit: 0 bytes in 0 blocks
==12796==   total heap usage: 2,497 allocs, 2,497 frees, 28,384 bytes allocated
==12796== 
==12796== All heap blocks were freed -- no leaks are possible
==12796== 
==12796== For lists of detected and suppressed errors, rerun with: -s
==12796== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
