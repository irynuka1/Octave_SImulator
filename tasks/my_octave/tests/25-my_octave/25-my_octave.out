8832 9670 2108 3028 
3981 1129 8887 4818 
6256 2177 2918 606 
5122 554 6772 9776 
4 4
779 7213 2564 476 3282 2350 8729 860 
5166 7040 4849 7116 1161 2073 1662 4112 
1683 5547 1699 9867 3229 949 7480 4304 
3912 7747 8888 2099 5421 1591 5734 9066 
9855 7285 918 573 3469 8490 962 9712 
3955 1663 984 8410 6417 674 4588 3621 
2586 1179 4009 3010 2925 8223 4094 4799 
8828 2217 5199 4042 7243 6024 8649 5110 
8 8
